docs/2.guide/2.directory-structure/1.components.md

```diff
@@ -119,6 +119,79 @@ const show = ref(false)
 </template>
 ```
 
+## Delayed Hydration
+
+In real world applications, some pages may include a lot of content and a lot of components, and most of the time not all of them need to be interactive as soon as the page is loaded. Having them all load eagerly can negatively impact performance and increase bundle size.
```

Comment:
It may be useful to outline the difference with Lazy prefixed components as it might seem like Lazy should solve this already.

Some rough prose, feel free to word however you like

> While Lazy prefixed components are useful for controlling the chunk size of your app, they won't necessarily improve the loading performance. This is because the async component is loaded eagerly with the rest of the page's scripts.

> Delayed hydration components fix this, allowing you to load the component just in time to maintain a good user experience while providing the performance benefit of not loading the component immediately.

Reply:
Rephrased it, let me know what you think!